.github/workflows/main.yaml

```diff
@@ -0,0 +1,345 @@
+on: 
+  push: 
+    tags:
+     - '*'
+
+jobs:
+  get-artifact-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: get the list
+        run: |
+         git clone https://github.com/amshinde/kata-packaging
+         pushd kata-packaging
+         tag=`echo $GITHUB_REF | cut -d/ -f3-`
+         git checkout $tag
+         popd
+         ./kata-packaging/artifact-list.sh > artifact-list.txt
+      - name: save-artifact-list
+        uses: actions/upload-artifact@master
+        with: 
+          name: artifact-list 
+          path: artifact-list.txt
+  build-kernel:
+    runs-on: ubuntu-16.04
```

Comment:
OOI, is this a list of distro-version combos? If so, could we add 18.04 too? And what about other distros?

Reply:
see https://help.github.com/en/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on